autoversioner/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM scratch
```

Comment:
can you put dockerfile in container_images/autoversioner directory?

Reply:
i actually started from there. and i spent an aweful amount of time integrating gobuild with cloudbuild and container image.
the [cloud-builders](https://github.com/GoogleCloudPlatform/cloud-builders) specifications are not very clear, even that did not help. the worse thing was i could not even build the example packages that cloud-builders team have in their repo. :(
I reached out the team through [issues](https://buganizer.corp.google.com/issues/142765191); at least i was able to get some pointers around how to proceed further.
That's when i was exposed to cloudbuild parameter called "dir" that lets you run go commands from a directory of your choice. After all experimentation; i wanted a working build for this package :rofl: 

I think, now i have a better understanding of how cloudbuild organizes the code to be built and what features we have at our disposal to achieve a build.

I should be able to move the cloudbuild configuration and container definition under container_images